radixdlt-java/src/test/java/com/radixdlt/client/core/pow/ProofOfWorkBuilderTest.java

```diff
@@ -33,4 +40,81 @@ public void test() throws ProofOfWorkException {
 
 		pow.validate();
 	}
+
+	@Test
+	public void test_vectors() {
```

Comment:
This does not conform to our styleguide, use `lowerCamelCase` for methods.

Reply:
sure, to be fair we allow snake case for descriptive tests "assert_that_x_do_y" etc. But will change this one!